src/components/map-control.tsx

```diff
@@ -8,35 +8,36 @@ type MapControlProps = PropsWithChildren<{
   position: ControlPosition;
 }>;
 
-/**
- * Copy of the `google.maps.ControlPosition` constants.
- * They have to be duplicated here since we can't wait for the maps API to load to be able to use them.
- */
 export enum ControlPosition {
```

Comment:
```suggestion
/**
 * Copy of the `google.maps.ControlPosition` constants.
 * They have to be duplicated here since we can't wait for the maps API to load to be able to use them.
 */
export enum ControlPosition {
```